src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,388 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Content Collections (Experimental)
+description: Content collections help organize your Markdown and type-check your frontmatter with schemas.
+i18nReady: false
+---
+
+Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. Collections may be helpful if you:
+
+- **Plan to use Markdown content in multiple areas** of your site (landing pages, footers, navigation, etc).
+- **Want Astro to enforce frontmatter fields,** and fail if fields are missing (e.g. every blog post should have a title and description).
+
+## Getting started
+
+Content Collections are experimental. To enable this feature, set the `experimental.contentCollections` flag in your Astro config:
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+  experimental: {
+    contentCollections: true,
+  },
+});
+```
+
```

Comment:
``````suggestion

You will also need to update `tsconfig.json`. Add `"strictNullChecks": true` under `compilerOptions`.

```json title="tsconfig.json" {4}
{
  "extends": "astro/tsconfigs/base"
  "compilerOptions": {
      "strictNullChecks": true
    },
}
```

``````
Edit as desired!

Reply:
Spaces is good 👍